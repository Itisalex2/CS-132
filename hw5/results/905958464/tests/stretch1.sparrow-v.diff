0a1,4
> 100
> 40464
> 80770
> 121302
