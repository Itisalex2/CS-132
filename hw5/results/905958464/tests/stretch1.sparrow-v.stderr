Translation Context:
TranslationContext{functionContexts={f=FunctionContext{functionName='f', parameters=[], localVariables=[a, b, c, x, y, d, f, e, fifty, s, t, forty, ten, twenty, z, w, two, seven, g, fiftyfive, h, v2, i, j, one, k, five, l, q, r, m, n, o, thirty, p, zero, u]}, g=FunctionContext{functionName='g', parameters=[x1, x2, x3, x4, x5, x6, x7, x8], localVariables=[a, b, c, d, ten, e, v3, v4, f, g, h, i, j]}, Main=FunctionContext{functionName='Main', parameters=[], localVariables=[v0, v1]}}}
